src/future/future/mod.rs

```diff
@@ -129,6 +133,90 @@ extension_trait! {
         {
             DelayFuture::new(self, dur)
         }
+
+        #[doc = r#"
+            Waits for one of two similarly-typed futures to complete.
+
+            Awaits multiple futures simultaneously, returning all results once complete.
```

Comment:
Is it "all results once complete" or "the result that completes first"?

Reply:
Good catch; it's the latter